evalml/tests/conftest.py

```diff
@@ -272,24 +272,48 @@ def regression_core_objectives():
 
 
 @pytest.fixture
-def stackable_classifiers():
+def stackable_classifiers(helper_functions):
     stackable_classifiers = []
     for estimator_class in _all_estimators():
         supported_problem_types = [handle_problem_types(pt) for pt in estimator_class.supported_problem_types]
         if (set(supported_problem_types) == {ProblemTypes.BINARY, ProblemTypes.MULTICLASS} and
             estimator_class.model_family not in _nonstackable_model_families and
                 estimator_class.model_family != ModelFamily.ENSEMBLE):
-            stackable_classifiers.append(estimator_class())
+            stackable_classifiers.append(helper_functions.safe_init_with_njobs_1(estimator_class))
     return stackable_classifiers
 
 
 @pytest.fixture
-def stackable_regressors():
+def stackable_regressors(helper_functions):
     stackable_regressors = []
     for estimator_class in _all_estimators():
         supported_problem_types = [handle_problem_types(pt) for pt in estimator_class.supported_problem_types]
         if (set(supported_problem_types) == {ProblemTypes.REGRESSION, ProblemTypes.TIME_SERIES_REGRESSION} and
             estimator_class.model_family not in _nonstackable_model_families and
                 estimator_class.model_family != ModelFamily.ENSEMBLE):
-            stackable_regressors.append(estimator_class())
+            stackable_regressors.append(helper_functions.safe_init_with_njobs_1(estimator_class))
     return stackable_regressors
+
+
+@pytest.fixture
+def helper_functions():
+    class Helpers:
+        @staticmethod
+        def safe_init_with_njobs_1(component_class):
+            try:
+                component = component_class(n_jobs=1)
+            except TypeError:
+                component = component_class()
+            return component
+
+        @staticmethod
+        def safe_init_pipeline_with_njobs_1(pipeline_class):
+            try:
+                estimator = pipeline_class.component_graph[-1]
+                estimator_name = estimator if isinstance(estimator, str) else estimator.name
+                pl = pipeline_class({estimator_name: {'n_jobs': 1}})
+            except ValueError:
+                pl = pipeline_class({})
+            return pl
```

Comment:
Looks good. It could mess with call counts if someone were mocking / introspecting on the component or pipeline constructor or something, but I don't think any of our current tests do that so nbd.

So we're not using this helper in places where we pass other parameters into the component or pipeline?

Reply:
Yep exactly. We have some tests that iterate over all estimators/pipelines, init with default parameters, and verify some functionality so we use these helpers to just change the `n_jobs` since it's not critical to what the test is verifying.